openhcl/virt_mshv_vtl/src/lib.rs

```diff
@@ -400,6 +400,16 @@ impl GuestVsmState {
             _ => None,
         }
     }
+
+    #[cfg_attr(guest_arch = "aarch64", allow(dead_code))]
+    fn get_hardware_cvm(&self) -> Option<&HardwareCvmVtl1State> {
```

Comment:
Still planning to refactor this into the backings, just wanted to get this change in first.